src/TodoApp.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import TodoList from './TodoList';
+import Footer from './Footer';
+
+class TodoApp extends React.Component {
+  state = {
+    todos: [],
+    inputValue: '',
+    allCompleted: false,
+    filterStatus: undefined,
+  }
+
+  addInputValue = (event) => {
+    this.setState({
+      inputValue: event.target.value,
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    if (!this.state.inputValue) {
+      return;
+    }
+
+    this.setState((state) => {
+      const newTodo = {
+        title: this.state.inputValue,
+        id: +new Date(),
+        completed: false,
+      };
+
+      return {
+        todos: [...state.todos, newTodo],
+        inputValue: '',
+      };
+    });
+  }
+
+  handleCompleted = (todoId) => {
+    this.setState(state => ({
+      todos: state.todos.map(todo => (todo.id === todoId
+        ? {
+          ...todo,
+          completed: !todo.completed,
+        }
+        : todo
+      )),
+    }));
+    this.setState(state => (
+      { allCompleted: state.todos.every(todo => todo.completed) }
+    ));
+  }
+
+  handleDelete = (todoId) => {
```

Comment:
пофікси назви методів, назви їх правильно, назва має відображати те що робить метод, в даному випадку:
```suggestion
  removeTodo = (todoId) => {
```